smtpd/mail.mboxfile.c

```diff
@@ -14,13 +14,24 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
+#include "config.h"
+
 #include <sys/types.h>
+
+#ifdef HAVE_SYS_STAT_H
```

Comment:
unsure about this.

either way, if there's a real reason to test sys/stat and do conditional inclusion, please do not reorder headers as this makes diffing against the non-portable version trickier.

The same comment applies to all headers that are reshuffled in this file.

Reply:
I will remove these.